include/clang/AST/Stmt.h

```diff
@@ -90,8 +90,12 @@ class alignas(void *) Stmt {
   class CompoundStmtBitfields {
     friend class CompoundStmt;
     unsigned : NumStmtBits;
+    // Is this compound statement checked or unchecked?
+    unsigned IsCheckedScope : 1;
+    // Was the property declared explicitly in the code?
+    unsigned CheckedPropertyDeclared : 1;
 
-    unsigned NumStmts : 32 - NumStmtBits;
+    unsigned NumStmts : 32 - 2 - NumStmtBits;
```

Comment:
Oh god. We really reducing the number of statements in a block to a quarter of its previous value? If this turns into an issue I'm going to be really sad.